mtpoe_ctrl.h

```diff
@@ -1,12 +1,14 @@
 #ifndef _MTPOE_CTRL
 #define _MTPOE_CTRL
 
+#include "mk_boards.h"
+
 //файл для общения с spidev модулем
 #define DEFAULT_DEV_FILE "/dev/spidev0.2"
 //ключ субконфига uci->network в котором находятся настройки PoE
-#define MTIK_POE_UCI_CONFIG_KEY "mtik_poe"
+#define MTIK_POE_UCI_CONFIG_KEY "@poe[0]"
```

Comment:
I needed to change this to just `poe`, otherwise `load_poe_from_uci` wouldn't find the config entries